app/assets/stylesheets/articles.scss

```diff
@@ -195,6 +195,11 @@
           margin-top:12px;
         }
       }
+      h4{
```

Comment:
I don't think this section is required, the lines added at 316 - 318 should be sufficient.